README.md

```diff
@@ -79,6 +79,65 @@ cd corral
 
 You can make a debug build with `.\make.ps1 build -Config Debug`
 
+## Getting started using Corral
+After instllation, make sure Corral's current path is also set in your PATH environment variable. This will make it easier by not entering the full path each time when you use one of Corral commands in your projects.
+
+Follow these steps to create your first project using Corral.
+
+1. Create an empty folder and switch to this directory. This will be our example project to use Corral
+    ```bash
+    mkdir myproject
+    cd myproject
+    ```
+
+2. Initialize Corral. It will create `corral.json` and `lock.json` files. At this moment they won't have much information since we haven't added any dependency yet.
+    ```bash
+    corral init
+    ```
+
+3. Add a dependency. This is the way to tell Corral that your project depends on this and you want to include it when building your project.
+    ```bash
+    corral add github.com/jemc/pony-inspect.git
+    ```
+
+4. Use that dependency in your code. Create a file `main.pony` with following code.
+    ```pony
+    use "inspect"
+      actor Main
+        new create(env: Env) =>
+          env.out.print(Inspect("Hello, World!"))
+    ```
+
+5. Fetch dependencies. As you can see in previous step that we are using `Inspect` type from the added depedency, we need to make sure that the code for this should be locally available on your machine so that we can compile without any errors.
+    ```bash
+    corral fetch
+    ```
+
+6. Build the project. Corral will now use all of the information which we have provide and try to build the project.
+    ```bash
+    corral run ponyc
+    ```
+
+If there are no errors generated then an executable `myproject` will be created in the same folder.
+
+If you prefer you can just copy the following code and run it in your favourite shell which will basically do the same which we have explained in previous steps.
```

Comment:
I think should remove "basically" from this. It suggests it will do something different. Which for end result, it won't.

That said, I would remove this section as it is of limited value in my mind and requires it to be updated in sync with the other code from above.

Reply:
I have removed it